### DISCUSSION TITLE
Negative linear collapsing boundary angle - Interpretation?

### DISCUSSION THREAD

**Discussion by jainraj** [lnccbrown/HSSM#549] [Q&A]
I fit a linear collapsing boundary model to experimental data in which participants have to respond within a maximum time (which they are aware of!). 

The estimated `theta` is coming to be close to -0.1 (lower bound for the LANs). I am wondering how to interpret this result. I was expecting some positive value. Should I try a non-collapsing version, then? 

PS: 
1. I am not using HSSM/HDDM - I use the LAN directly. (The question still stands - how to interpret negative angle). 
2. LAN - Race model with 4 choices, same starting point across all choices with linear collapsing boundary.

**Comment by frankmj** [comment]
Hi, probably this shouldn;'t be an HSSM discussion as there are many
reasons one might obtain some fits to a model depending on the data and the
model set up and the inference method so it is pretty hard to diagnose.

That said:

-  the usual strategies are to check that you can recover ground
truth parameters from simulated data with your method, and that when you
fit the data you can reproduce core features of it  (RT quanitles, error
rates) with your fitted parameters

- note that race models give you a kind of urgency for free without
requiring collapsing bounds - if all accumulators have some constant
intercept on drift then these will continue to rise throughout the course
of a trial toward the boundary independent of the evidence - so that any
impact of noise will be greater with more time, similar to collapsing
bounds.. in contrast when using a DDM  the drift rate accumulates the
difference in evidence between options and so does not have this same
property unless you had a collapsing bound. So when you have a race model
you might not need a collapsing bound to account for the data (and hitting
the negative boundary might be trying to compensate for the effect I
described above).

**Reply by jainraj** [reply]
Thank you, Professor, for the suggestion and the interpretation.
